tests/Browser/console/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
What's this folder used for?

Reply:
@LukeTowers Dusk will print out error logs in that folder when running Dusk tests. It's similar to the screenshots subfolder in the same folder.